scripts/prepare_env_windows.ps1

```diff
@@ -0,0 +1,40 @@
+$PACKAGES= @{ git = ""; golang = "1.18.3"; make = "" }
+
+Write-Host "Downloading chocolatey package"
+curl.exe -L "https://packages.chocolatey.org/chocolatey.0.10.15.nupkg" -o 'c:\choco.zip'
+Expand-Archive "c:\choco.zip" -DestinationPath "c:\choco"
+
+Write-Host "Installing choco"
+& "c:\choco\tools\chocolateyInstall.ps1"
+
+Write-Host "Set choco.exe path."
+$env:PATH+=";C:\ProgramData\chocolatey\bin"
+
+Write-Host "Install necessary packages"
+
+foreach ($package in $PACKAGES.Keys) {
+    $command = "choco.exe install $package --yes"
+    $version = $PACKAGES[$package]
+    if (-Not [string]::IsNullOrEmpty($version)) {
+        $command += " --version $version"
+    }
+    Invoke-Expression $command
+}
+
+Write-Host "Set up environment."
+
+$userGoBin = "${env:HOME}\go\bin"
+$path = ";c:\Program Files\Git\bin;c:\Program Files\Go\bin;${userGoBin};"
+$env:PATH+=$path
+
+Write-Host $env:PATH
+
+[Environment]::SetEnvironmentVariable("PATH", $env:PATH, 'User')
+
+# Prepare Log dir
+mkdir c:\Logs
+
+# Log go env for future reference:
+go env > c:\Logs\go-env.txt
+cat c:\Logs\go-env.txt
```

Comment:
These are not collected, since the command in the bash script is ``copy_from 'c:/Logs/*.xml' ${ARTIFACTS} ${VM_PUB_IP}``.

Reply:
Not collected, I just want it in the build log for visibility. Can remove the file tho.